src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -70,14 +70,15 @@
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.String.cs" />
     <Compile Include="System\Text\Json\Writer\Utf8JsonWriter.WriteValues.UnsignedNumber.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
-    <ProjectReference Include="..\..\System.Memory\src\System.Memory.csproj" />
-    <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj" />
+  <ItemGroup Condition="'$(TargetsNETStandard)' != 'true'">
+    <Reference Include="System.Diagnostics.Debug" />
+    <Reference Include="System.Resources.ResourceManager" />
+    <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.Extensions" />
+    <Reference Include="System.Text.Encoding.Extensions" />
+    <Reference Include="System.Threading.Tasks" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard'">
-    <ReferenceFromRuntime Include="System.Private.CoreLib" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
```

Comment:
Why was this condition now unnecessary?

Reply:
Because everything is compiling against the public reference types.  Previously you had a configuration that compiled against implementations so they needed project references with typeforwards that would unify System.Private.CoreLib.  Now everything is using public reference types so the only real difference between netstandard and NETCoreApp (modulo the netstandard.dll reference) is that the NETCoreApp config needs individual references for each assembly that makes up netstandard surface (above).